package.json

```diff
@@ -14,7 +14,7 @@
     "@mui/styled-engine-sc": "^5.12.0",
     "@mui/styles": "^5.11.9",
     "@reduxjs/toolkit": "^1.9.3",
-    "@terraware/web-components": "^2.3.45",
+    "@terraware/web-components": "^2.3.46",
```

Comment:
i have a PR out with version 47, we probably need to coordinate that

Reply:
I guess whichever one goes in first will cause a merge conflict on the other, that's easy to resolve 🤷 How would you like to coordinate further than this?

Reply:
Doesn't matter. You can merge yours. I just approved it, and it's simpler and smaller.